src/python/install.sh

```diff
@@ -865,6 +865,12 @@ if [ "${INSTALL_JUPYTERLAB}" = "true" ]; then
     install_user_package $INSTALL_UNDER_ROOT jupyterlab
     install_user_package $INSTALL_UNDER_ROOT jupyterlab-git
 
+    if [ "$INSTALL_UNDER_ROOT" = false ]; then
+        # JupyterLab would have installed into /home/${USERNAME}/.local/bin
+        # Adding it to default path for Codespaces which use non-login shells
+        echo "Defaults secure_path=\"/home/${USERNAME}/.local/bin:${PATH}\"" >> /etc/sudoers.d/$USERNAME
```

Comment:
Rethinking about this one, if you have multiple `Defaults secure_path` lines under /etc/sudoers.d/, the last one encountered will be the one that takes effect. This is because each `Defaults secure_path` line encountered will overwrite the previous one.

Hence, if you want to add to the `secure_path`, you need to check if this entry exists, is so then you would need to first extract the current `secure_path`, add your new path to it, and then write it back.

Something like this 👇 ( this is not tested, so would be good to test it)

```
# Define the new path
new_path="/home/${USERNAME}/.local/bin"

# Check if the file contains "Defaults secure_path"
if sudo grep -q "Defaults secure_path" /etc/sudoers.d/$USERNAME; then
  # Extract the current secure_path
  current_path=$(sudo grep secure_path /etc/sudoers.d/$USERNAME | sed -e 's/Defaults secure_path="//' -e 's/"//')

  # Add the new path
  secure_path="${current_path}:${new_path}"

  # Replace the secure_path in the sudoers file
  sudo sed -i -e "s|Defaults secure_path=\"${current_path}\"|Defaults secure_path=\"${secure_path}\"|" /etc/sudoers.d/$USERNAME
else
  # If "Defaults secure_path" doesn't exist in the file, append it
  echo "Defaults secure_path=\"${new_path}:${PATH}\"" >> /etc/sudoers.d/$USERNAME
fi
```